www/js/recommendations.js

```diff
@@ -6,15 +6,19 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
                 'emission.splash.referral', 'angularLocalStorage',
                 'ng-walkthrough', 'nzTour'])
 
+/*
+Ask Shankari about this Wednesday
```

Comment:
why wait until Wednesday!?

Reply:
not every discussion needs to happen in person. In fact, for a lot of open source projects, no discussions will happen in person.